### ISSUE TITLE
The num_heads of CSWin-B in the paper and code are not the same

### ISSUE THREAD

**Issue by 282857341** [microsoft/CSWin-Transformer#18]
Hi!

Your work is great. But I’m confused with the num_heads in the CSWin-B.
In the cswin.py, the num heads is [4,8,16,32]
https://github.com/microsoft/CSWin-Transformer/blob/d8be74a7833898f7bd9c77eb8c051d1b8bd5d753/models/cswin.py#L392

But in your paper, the num heads of the CSWin-B  is [2, 4, 8, 16].

**Comment by LUO77123** [comment]
CSWin-B 96 2,4,32,2 1,2,7,7 4,8,16,32 78M 15.0G是一样的啊

**Comment by wujiang0156** [comment]
@282857341 yes ,  in the paper, the num heads of the CSWin-B is [2, 4, 8, 16]. but In the cswin.py, the num heads is [4,8,16,32], why ? which is right ,please
